Cargo.toml

```
@@ -59,6 +58,7 @@ alloc = [
   "futures-core/alloc",
   "pin-project-lite",
 ]
+tokio = ["async-global-executor/tokio"]
```

I'd prefer we add a version qualifier to this as well. There are a lot of language features and traits missing in Rust right now which will almost certainly require breaking changes in Tokio. This is the same reality async-std has operated in since our 1.0 release. Ensuring we're prepared for the eventual Tokio 2.0 release would be wise.

I'd think something like:
```suggestion
tokio1 = ["async-global-executor/tokio"]
```

---

Sure, I was thinking of `tokio` for 1.0 and `tokio2` for 2.0, but I can make the former `tokio1`